test/sanitizer_suppressions/lsan

```diff
@@ -0,0 +1,7 @@
+leak:addCoin
```

Comment:
Could add a commit with some documentation about those (see my tsan file in this folder for example):

* `addCoin` is some bench test that doesn't primarily care about leaks.
* The lib* ones are likely upstream false-positives
* ...